### ISSUE TITLE
Use existing module for xml building?

### ISSUE THREAD

**Issue by gmaclennan** [digidem/osm-p2p-xml#1]
Why not [node-xml](https://github.com/dylang/node-xml) or other existing xml builder rather than implementing our own? Better testing coverage and more edge-cases probably already dealt with. Same for https://github.com/digidem/osm-p2p-server/blob/master/lib/h.js

**Comment by nyurik** [comment]
I agree, this seems like a potential for all sorts of small and hard to notice errors.  I really want to use this lib for my user-facing osm project, but it would be a bigger risk, both stability and security-wise.  BTW (unrelated) - if possible, please include an example of parsing OSM API requests/responses. I suspect they should be fairly similar to file parsing.  Thanks!!!

P.S.  https://github.com/toutpt/angular-osm seems to have used a very good approach for this
